src/metatensor/models/utils/data/readers/readers.py

```diff
@@ -166,10 +166,24 @@ def read_targets(
     :param conf: config containing the keys for what should be read.
     :param dtype: desired data type of returned tensor
     :returns: Dictionary containing one TensorMaps for each target section in the
-        config."""
+        config.
+
+    :raises ValueError: if the target name is not valid. Valid target names are
+        those that either start with ``mtm::`` or those that are in the list of
+        standard outputs of ``metatensor.torch.atomistic`` (see
+        https://docs.metatensor.org/latest/atomistic/outputs.html)
+    """
     target_dictionary = {}
+    standard_outputs_list = ["energy"]
 
     for target_key, target in conf.items():
+        if target_key not in standard_outputs_list and not target_key.startswith(
+            "mtm::"
+        ):
+            raise ValueError(
+                f"Target names must either be one of {standard_outputs_list} "
+                "or start with `mtm::`."
```

Comment:
Please add a test that this error is raised correctly.

Reply:
Done!